package.json

```diff
@@ -74,7 +80,6 @@
         "karma-coverage-istanbul-reporter": "3.0.3",
         "karma-jasmine": "4.0.1",
         "karma-jasmine-html-reporter": "1.7.0",
-        "lerna": "6.6.1",
```

Comment:
Wasn't there some config file for lerna? Should we remove it too?

Reply:
fixed